RadarSDK/RadarVerificationManager.m

```diff
@@ -31,6 +45,144 @@ + (instancetype)sharedInstance {
+                                         [RadarUtils runOnMainThread:^{
+                                             completionHandler(status, nil);
+                                         }];
+                                     }
+
+                                     return;
+                                 }
+
+                                 [self getAttestationWithNonce:config.nonce
+                                             completionHandler:^(NSString *_Nullable attestationString, NSString *_Nullable keyId, NSString *_Nullable attestationError) {
+                                                 [[RadarAPIClient sharedInstance]
+                                                     trackWithLocation:location
+                                                               stopped:RadarState.stopped
+                                                            foreground:[RadarUtils foreground]
+                                                                source:RadarLocationSourceForegroundLocation
+                                                              replayed:NO
+                                                               beacons:nil
+                                                              verified:YES
+                                                     attestationString:attestationString
+                                                                 keyId:keyId
+                                                      attestationError:attestationError
+                                                             encrypted:YES
+                                                     completionHandler:^(RadarStatus status, NSDictionary *_Nullable res, NSArray<RadarEvent *> *_Nullable events,
+                                                                         RadarUser *_Nullable user, NSArray<RadarGeofence *> *_Nullable nearbyGeofences,
+                                                                         RadarConfig *_Nullable config, NSString *_Nullable token) {
+                                                         if (status == RadarStatusSuccess && config != nil) {
+                                                             [[RadarLocationManager sharedInstance] updateTrackingFromMeta:config.meta];
+                                                         }
+                                                         if (completionHandler) {
+                                                             [RadarUtils runOnMainThread:^{
+                                                                 completionHandler(status, token);
+                                                             }];
+                                                         }
+                                                     }];
+                                             }];
+                             }];
+        }];
+}
+
+- (void)startTrackingVerified:(BOOL)token {
```

Comment:
We're not presently `[RadarSettings setTracking:YES];` so `[Radar isTracking`] = NO;`.

IDK yet the best way of having incorporating / handling ordinary tracking with verified tracking. Maybe we want a separate bool?

Also we probably need a `stopTracking`?

Reply:
I think this needs to operate totally separately from `startTracking()`. Most common usage pattern is going to be to start this on app open / login and then never stop it. For now, if you want more fine-grained control, you can call `trackVerified()` manually. We can always add `stopTrackingVerified()` later if needed